lib/IRGen/GenDiffFunc.cpp

```diff
@@ -32,6 +32,7 @@
 using namespace swift;
 using namespace irgen;
 
+/// A pair of `@differentiable` function extractee and differentiation order.
```

Comment:
Is this related to this change?

Reply:
No, but I felt it was okay to add a not-exactly-related doc comment.

I added this comment because it clarifies `fields.push_back(std::make_pair(AutoDiffFunctionExtractInst::Extractee::VJP, 1));` in [`TypeConverter::convertDifferentiableFunctionType`](https://github.com/apple/swift/blob/51dc28d6266d21e2f27c4c600a1f35f32654d069/lib/IRGen/GenDiffFunc.cpp#L185). We came across this while debugging the IRGenDebugInfo crash.